projects/plugins/jetpack/modules/custom-post-types/portfolios.php

```diff
@@ -396,107 +460,145 @@ function updated_messages( $messages ) {
 	/**
 	 * Change ‘Title’ column label
 	 * Add Featured Image column
+	 *
+	 * @param array $columns An array of column names.
 	 */
-	function edit_admin_columns( $columns ) {
-		// change 'Title' to 'Project'
+	public function edit_admin_columns( $columns ) {
+		// change 'Title' to 'Project'.
 		$columns['title'] = __( 'Project', 'jetpack' );
 		if ( current_theme_supports( 'post-thumbnails' ) ) {
-			// add featured image before 'Project'
-			$columns = array_slice( $columns, 0, 1, true ) + array( 'thumbnail' => '' ) + array_slice( $columns, 1, NULL, true );
+			// add featured image before 'Project'.
+			$columns = array_slice( $columns, 0, 1, true ) + array( 'thumbnail' => '' ) + array_slice( $columns, 1, null, true );
 		}
 
 		return $columns;
 	}
 
 	/**
 	 * Add featured image to column
+	 *
+	 * @param string $column  The name of the column to display.
+	 * @param int    $post_id The current post ID.
 	 */
-	function image_column( $column, $post_id ) {
-		global $post;
-		switch ( $column ) {
-			case 'thumbnail':
-				echo get_the_post_thumbnail( $post_id, 'jetpack-portfolio-admin-thumb' );
-				break;
+	public function image_column( $column, $post_id ) {
+		if ( 'thumbnail' !== $column ) {
+			return;
 		}
+
+		echo get_the_post_thumbnail( $post_id, 'jetpack-portfolio-admin-thumb' );
```

Comment:
This felt cleaner to me than the switch statement with only one case, but I'm happy to be convinced otherwise.

Reply:
Agree!